src/scripts/main.js

```diff
@@ -1,8 +1,29 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
+const information = ['name', 'sex', 'born', 'died', 'age', 'century'];
```

Comment:
Якщо ці змінні залишиш то назви точно треба змінити бо не дуже ясно що за інформація, де розташована
```suggestion
const columns= ['name', 'sex', 'born', 'died', 'age', 'century'];
```
columns or keys